go/host/enclave/state.go

```diff
@@ -40,6 +40,9 @@ const (
 	L2Catchup
 )
 
+// when the L2 head is 0 then it means no batch has been seen or processed (first seq number is always 1)
```

Comment:
General comment regarding this "StateTracker" typte (unrelated to this change):

There are these 2 fields:
```
-	status Status
-	enclaveStatusCode common.StatusCode
```

quite confusing to understand the difference between them.

If I understand correctly: "status" is the state of the "StateTracker" and "enclaveStatusCode" is the internal state of the enclave ( as per its last reporting)

Reply:
yeah that's right, I've just updated the comments to try and make that clearer but agree it's a bit confusing atm